tests/test_filter.py

```diff
@@ -38,3 +38,26 @@ def test_language_pickleable(obj):
 )
 def test_language_deepcopyable(obj):
     assert obj == copy.deepcopy(obj)
+
+
+@pytest.mark.parametrize('obj,obj_inverted', [
+    (Languages.in_(['EN', 'RU']), Languages.not_in(['EN', 'RU'])),
+    (Skill('123') == 10, Skill('123') != 10),
+    (Skill('123') > 10, Skill('123') <= 10),
+])
+def test_filter_invertion(obj, obj_inverted):
+    assert ~obj == obj_inverted
+    
+
+@pytest.mark.parametrize('obj,obj_inverted', [
+    (
+        Languages.in_(['EN', 'RU']) & (Skill('123') == 10),
+        Languages.not_in(['EN', 'RU']) | (Skill('123') != 10)
+    ),
+    (
+        Languages.in_(['RU']) | ((Skill('123') > 10) & (Skill('123') <= 90)),
+        Languages.not_in(['RU']) & ((Skill('123') <= 10) | (Skill('123') > 90)),
+    ),
+])
```

Comment:
Why don't we combine this test with previous?

Reply:
Because they test different logic: the previous tests only inversion of simple `Condition`; this tests inversion of complex filters with usage of `&` and `|`.